wren-modeling-rs/core/src/logical_plan/rule.rs

```diff
@@ -170,6 +149,40 @@ impl ModelAnalyzeRule {
     }
 }
 
+fn belong_to_mdl(mdl: &WrenMDL, table_reference: TableReference) -> bool {
+    let catalog_mismatch = table_reference
+        .catalog()
+        .map_or(true, |c| c != mdl.catalog());
+
+    let schema_mismatch = table_reference.schema().map_or(true, |s| s != mdl.schema());
+
+    catalog_mismatch || schema_mismatch
+}
```

Comment:
IMO, using positive conditions will be better than negative. How about using `catalog_is_match` && `schema_is_match`?

Reply:
hmm I agree with you. However, after I tried to change to positive condition, it triggers another qualified name bugs that it's related what I'm fixing. I think I can fix them together in next PR. Let's merge this first.